setup.py

```diff
@@ -28,7 +28,7 @@
     "numpy>=1.21.5",
     "appdirs>=1.4.0",
     "pandas>=1.1.5",
-    "h5py>=3.7.0",
+    "h5py>=3.8.0",
```

Comment:
Earlier there was an issue while installing h5py via pip in python 3.11 which used to build from h5py source. Now python 3.11 wheels have been published with h5py version 3.8 (https://github.com/h5py/h5py/issues/2146), that issue has been resolved.